wrappers/obj-c/ODWLogConfiguration.h

```diff
@@ -4,111 +4,425 @@
+ @brief Return a copy of the default configuration
+*/
++(nullable ODWLogConfiguration *)getLogConfigurationCopy;
+
+/*!
+ @brief Sets the URI of the event collector.
+ @param eventCollectorUri A string for event collector uri.
 */
 +(void)setEventCollectorUri:(nonnull NSString *)eventCollectorUri;
 
 /*!
-@brief Returns the URI of the event collector.
+ @brief Returns the URI of the event collector.
 */
 +(nullable NSString *)eventCollectorUri;
 
 /*!
-@brief Sets the RAM queue size limit in bytes.
-@param cacheMemorySizeLimitInBytes  A long value for memory size limit in bytes.
+ @brief Sets the RAM queue size limit in bytes.
+ @param cacheMemorySizeLimitInBytes  A long value for memory size limit in bytes.
 */
 +(void)setCacheMemorySizeLimitInBytes:(uint64_t)cacheMemorySizeLimitInBytes;
 
 /*!
-@brief return the RAM queue size limit in bytes.
+ @brief return the RAM queue size limit in bytes.
 */
 +(uint64_t)cacheMemorySizeLimitInBytes;
 
 /*!
-@brief Sets the size limit of the disk file used to cache events on the client side.
-@param cacheFileSizeLimitInBytes  A long value for cache file size limit.
+ @brief Sets the size limit of the disk file used to cache events on the client side.
+ @param cacheFileSizeLimitInBytes  A long value for cache file size limit.
 */
 +(void)setCacheFileSizeLimitInBytes:(uint64_t)cacheFileSizeLimitInBytes;
 
 /*!
-@brief Returns the size limit of the disk file used to cache events on the client side.
+ @brief Returns the size limit of the disk file used to cache events on the client side.
 */
 +(uint64_t)cacheFileSizeLimitInBytes;
 
 /*!
-@brief Sets max teardown upload time in seconds.
-@param maxTeardownUploadTimeInSec An integer that time in seconds.
+ @brief Sets max teardown upload time in seconds.
+ @param maxTeardownUploadTimeInSec An integer that time in seconds.
 */
 +(void)setMaxTeardownUploadTimeInSec:(int)maxTeardownUploadTimeInSec;
 
 /*!
-@brief Sets if trace logging to file is enabled.
-@param enableTrace True if tracing is enabled.
+ @brief Sets if trace logging to file is enabled.
+ @param enableTrace True if tracing is enabled.
 */
 +(void)setEnableTrace:(bool)enableTrace;
 
 /*!
-@brief Sets if console logging from the iOS wrapper is enabled.
-@param enableConsoleLogging True if logging is enabled.
+ @brief Sets if console logging from the iOS wrapper is enabled.
+ @param enableConsoleLogging True if logging is enabled.
 */
 +(void)setEnableConsoleLogging:(bool)enableConsoleLogging;
 
 /*!
-@brief Sets the internal SDK debugging trace level.
-@param TraceLevel one of the ACTTraceLevel values.
+ @brief Sets the internal SDK debugging trace level.
+ @param TraceLevel one of the ACTTraceLevel values.
 */
 +(void)setTraceLevel:(int)TraceLevel;
 
 /*!
-@brief Returns true if tracing is enabled.
+ @brief Returns true if tracing is enabled.
 */
 +(bool)enableTrace;
 
 /*!
-@brief Returns true if console logging is enabled.
+ @brief Returns true if console logging is enabled.
```

Comment:
nit: please revert extra space on all these comments

Reply:
Spaces are still there, same comment applies to other files in this PR. I know this might not have on execution, but let's keep change size smaller

Reply:
Hi, you are looking at an OUTDATED commit, please take a look at latest commit https://github.com/microsoft/cpp_client_telemetry/pull/785/files